spec/support/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{

```

Comment:
is this config needed? we have spec/tsconfig.json already

Reply:
Same answer as above. Paths are `dist/src` for the tests and only for the tests.